GG/COPYING

```diff
@@ -1,16 +1,16 @@
-		  GNU LESSER GENERAL PUBLIC LICENSE
-		       Version 2.1, February 1999
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Why the tabs -> spaces? Was the license text changed, or originally written that way?